Chat/Defines.h

```diff
@@ -0,0 +1,23 @@
+#ifndef DEFINES_H //pragma once for losers
+#define DEFINES_H
+
+//default port
+#define PORT (unsigned short)10062
```

Comment:
It is good. =)
You don't write 10062 everywhere. So there is no magic numbers. If you have some constants, please define them as here or count by yourselves.